src/world/area_hos/hos_05/A2AAC0.c

```diff
@@ -196,13 +198,52 @@ ApiStatus func_80240E30_A2B070(Evt* script, s32 isInitialCall) {
 }
 
 // adjusts properties of EmitterVolume:GoldShimmer2 effect
-INCLUDE_ASM(s32, "world/area_hos/hos_05/A2AAC0", func_80240E50_A2B090);
+ApiStatus func_80240E50_A2B090(Evt* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 posX;
+    s32 posY;
+    f32 posZ;
+    EffectInstance* effect;
+
+    effect = (EffectInstance*) evt_get_variable(script, *args++);
```

Comment:
Could you move this under the args initialization and initialize it on the same line you declare it?